src/lib/components/profile/modal/Modal.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import cancelIcon from '$lib/icons/cancel-black.svg'
+  import { fade, scale } from 'svelte/transition'
+
+  export let isOpen: Boolean
```

Comment:
```suggestion
  export let isOpen: boolean
```
El linter falló aquí